src/plugins/ovp/components/livestreams-screen/LivestreamsCard.jsx

```diff
@@ -108,23 +108,33 @@ const LivestreamsCard = ({ nodes, metas, reloadChannelState }) => nodes.map((nod
         .set('node_ref', nodeRef);
       const pbjx = await app.getPbjx();
       await pbjx.send(command);
-      await delay(5000);
-      reloadChannelState();
-      await progressIndicator.close();
-      await delay(2000);
-      dispatch(sendAlert({
-       type: 'success',
-       isDismissible: true,
-       delay: 3000,
-       message: 'Success! The MediaLive Channel was started.',
-      }));
-    }catch(error){
-      await progressIndicator.close();
+      const worker = app.getParameter('raven.worker');
```

Comment:
@meghedi you can subscribe to the pbjx event listener rather than going through worker.  the worker (should be) sending the pbjx events to the app.  we want to minimize any direct listening to workers outside of raven, which is the service handling that interaction.